README.rst

```diff
@@ -3,23 +3,36 @@
 Motr
 =====
 
-.. Slack Badge  - https://img.shields.io/badge/chat-on%20Slack-blu
-
-.. image:: https://img.shields.io/badge/chat-on%20Slack-blue
-        :target: https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?
-        :alt: Slack chat badge
+.. License Badge - https://img.shields.io/badge/License-Apache%202.0-blue.svg
 
+.. image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+        :target: https://github.com/pujamudaliar/cortx-motr/blob/main/LICENCE
```

Comment:
Change https://github.com/pujamudaliar/cortx-motr/blob/main/LICENCE

to https://github.com/Seagate/cortx-motr/blob/main/LICENCE

Reply:
yes. I did not notice this.

Reply:
Fixed already